.travis.yml

```diff
@@ -37,21 +37,22 @@ before_script:
 jobs:
   include:
     - stage: build
+      os: linux
       script:
           - make build
           - make extracheck
-      os: linux
     - stage: build
-      script: make build
       os: osx
+      script: make build
     - stage: test
-      script: true
       os: osx
+      install: make install
+      script: cd ./travis-scripts/ && ./test-mac.sh
```

Comment:
Moved to running only the mender-artifact read,write validate functionality on mac.

Reply:
This folder is only for Mac testing, the fact that is in Travis is just an implementation detail (that eventually we will change). I suggest just naming the folder `tests-accpentance-mac` or similar